build_config/README.md

```diff
@@ -123,6 +124,57 @@ builders:
     auto_apply: dependents
 ```
 
+## Defining `PostProcessBuilder`s
+
+`PostProcessBuilder`s are configured similarly to normal `Builder`s, but they
+have some different/missing options.
+
+These builders can not be auto-applied on their own, and must always build to
+cache because their outputs are not declared ahead of time. To apply them a
+user will need to explicitly enable them on a target, or a `Builder` definition
+can add them to `apply_builders`.
+
+Exposed `PostProcessBuilder`s are configured in the `post_process_builders`
+section of the  `build.yaml`. This is a map of builder names to configuration.
+Each post process builder config may contain the following keys:
+
+- **target**: The name of the target which defines contains the `Builder` class
+  definition.
+- **import**: Required. The import uri that should be used to import the library
+  containing the `Builder` class. This should always be a `package:` uri.
+- **builder_factories**: A `List<String>` which contains the names of the
+  top-level methods in the imported library which are a function fitting the
+  typedef `Builder factoryName(List<String> args)`.
+- **input_extensions**: Required. A list of input extensions that will be
+  processed. This must match the merged `inputExtensions` lists from each
+  `Builder` in `builder_factories`.
```

Comment:
`PostProcessBuilder` instead of `Builder` and can rephrase to remove the idea of merging.

Reply:
done